documentation/docs/10-getting-started/30-project-structure.md

```diff
@@ -53,13 +53,15 @@ The `src` directory contains the meat of your project.
 
 You can use `.ts` files instead of `.js` files, if using TypeScript.
 
+If you added [Vitest](https://vitest.dev) when you set up your project, your unit tests will live in the `src` directory with a `.test.js` (or `.test.ts`) extension.
```

Comment:
According to vite.config.js the infix can also be `.spec.` not necesserarily test.